configs/mmpose/yolox-pose_onnxruntime_static.py

```diff
@@ -0,0 +1,22 @@
+_base_ = ['./pose-detection_static.py', '../_base_/backends/onnxruntime.py']
```

Comment:
The filename must follow certain rules. Pls. check [this ](https://mmdeploy.readthedocs.io/en/latest/02-how-to-run/write_config.html) doc. The name could be 
`configs/mmpose/pose-detection_yoloxpose_onnxruntime_dynamic.py`